cachito/web/api_v1.py

```diff
@@ -29,12 +33,26 @@ def get_requests():
         value exceeds configuration's CACHITO_MAX_PER_PAGE.
     :rtype: flask.Response
     """
-    # Default versbose flag to False
+    # Check if user is filtering requests by state
+    state = flask.request.args.get('state', default=None)
```

Comment:
Could you please document this query parameter in the OpenAPI document at `cachito/web/static/api_v1.yaml`? It's how we generate our API documentation at https://release-engineering.github.io/cachito.

Here is the documentation: https://swagger.io/docs/specification/describing-parameters/#query-parameters